client/src/components/categorical/value.js

```diff
@@ -34,14 +36,23 @@ class CategoryValue extends React.Component {
       value,
       colorAccessor,
       colorScale,
-      i
+      i,
+      schema
     } = this.props;
 
     if (!categoricalAsBooleansMap) return null;
 
     const selected = categoricalAsBooleansMap[metadataField][value];
     /* this is the color scale, so add swatches below */
     const c = metadataField === colorAccessor;
+    let categories = null;
+
+    if (c && schema) {
+      /* lint wants [categories] which seems less clear, prefer [0] over array destructuring */
```

Comment:
we should configure eslint to what we want and then roll it out globally (rather than letting it be ad hoc).  I personally am OK with either, with a very slight preference for destructuring.  But I'm OK if you want to standardize on not using it.